tests/josegonzalez/Dotenv/LoaderTest.php

```
@@ -141,6 +141,7 @@ public function testParse()
             'STRING_FALSE' => 'false',
             'STRING_EMPTY' => '',
             'STRING_EMPTY_2' => '',
+            '01SKIPPED'=>'skipped',
```

Fixed, but instead of ignoring it throws an exception